cmd/aws-cloud-controller-manager/main.go

```diff
@@ -78,7 +78,7 @@ func main() {
 	app.ControllersDisabledByDefault.Insert(tagging.TaggingControllerKey)
 
 	controllerAliases := names.CCMControllerAliases()
-	controllerAliases[tagging.TaggingControllerKey] = tagging.TaggingControllerClientName
+	controllerAliases[tagging.TaggingControllerKey] = tagging.TaggingControllerKey
```

Comment:
Hi @chewong , regarding 1.28 KCM and CCM new feature on controller alias,  if we have some customized controller implemented in the ccm provider, should we also update its name convention to same pattern? I guess no but still want to double confirm: If we don't do this work, will it impact any functionality like logging etc.,?  Thanks.